### PR TITLE
#209 Organisms - FOCardWithTextField

### PR DESCRIPTION
**Pull request by bryantobing12**
### Detailed description of changes

- Need to update the component to match the design

### Recommendations on how to test the changes

- Check if the component match the design

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

### Link to related issue(s)

https://app.zenhub.com/workspaces/flick-614155581501e1000f356a5d/issues/hzn-one/tms/209

#### Before

N / A

#### After

image

### CODE REVIEW

**Review by didikmulyadi — APPROVED** [review]

**Review by didikmulyadi — APPROVED** [review]
dari code OK, kesesuain UI tunggu yg lain yaa

File: src/Components/Organisms/FOCardWithTextField/index.tsx [hunk 1]
```diff
@@ -26,8 +26,9 @@ export const FOCardWithTextField = memo((props: IFOCardWithTextField) => {
       data-testid={testID}
       {...materialUIProps}
       sx={{
-        ...sx,
+        maxWidth: 560,
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOCardWithTextField/index.tsx:29]: set maxWidth better di wrapper project yg pakai component ini.

File: src/Components/Organisms/FOCardWithTextField/index.tsx [hunk 2]
```diff
@@ -26,8 +26,9 @@ export const FOCardWithTextField = memo((props: IFOCardWithTextField) => {
...
         boxShadow: "0px 0px 16px rgba(0, 0, 0, 0.1)",
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOCardWithTextField/index.tsx:30]: ini coba pakai props elevation, kalo value nya belum ada bisa ditambah di baseTheme

File: src/Components/Organisms/FOCardWithTextField/index.tsx [hunk 3]
```diff
@@ -47,7 +48,7 @@ export const FOCardWithTextField = memo((props: IFOCardWithTextField) => {
           variant="body1"
           letterSpacing="0.2px"
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOCardWithTextField/index.tsx:49]: better di remove letterSpacing nya, karena harusnya sudah di set di variant typography yg dipilih

Comment (gilangdarmawan) [reply src/Components/Organisms/FOCardWithTextField/index.tsx:49]: [di component FA Button](https://github.com/HZN-one/flex/blob/8c8e062b8ccee536929079aa655e3178ae6ddabb/src/Components/Organisms/FOCardWithTextField/index.tsx#L63) ga boleh ada `sx` berkaitan dengan background color dan border radius. untuk color bisa pakai props `color` dan border-radius sudah di set di baseTheme-nya button

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

File: src/Components/Organisms/FOCardWithTextField/index.tsx (hunk 3 above)
Comment (gilangdarmawan) [reply src/Components/Organisms/FOCardWithTextField/index.tsx:49]: di [FATypography](https://github.com/HZN-one/flex/blob/8c8e062b8ccee536929079aa655e3178ae6ddabb/src/Components/Organisms/FOCardWithTextField/index.tsx#L39) tidak disarankan set fontWeight lagi karena sudah di set di variant `component` h3 nya

Comment (bryantobing12) [reply src/Components/Organisms/FOCardWithTextField/index.tsx]: updated @gilangdarmawan

**Review by gilangdarmawan — APPROVED** [review]

**Pull request merged**